Library/Homebrew/diagnostic.rb

```diff
@@ -595,12 +596,17 @@ def check_coretap_git_origin
 
             Unless you have compelling reasons, consider setting the
             origin remote to point at the main repository by running:
-              git -C "#{coretap_path}" remote set-url origin #{Formatter.url("https://github.com/Homebrew/homebrew-core.git")}
+              git -C "#{coretap_path}" remote set-url origin #{Formatter.url(default_remote)}
           EOS
         end
+      end
 
+      def check_coretap_git_branch
         return if ENV["CI"]
 
+        coretap_path = CoreTap.instance.path
+        return if !Utils.git_available? || !(coretap_path/".git").exist?
+
         branch = coretap_path.git_branch
         return if branch.nil? || branch =~ /master/
 
```

Comment:
Want to update the string two lines below, too?

Reply:
You mean change `Homebrew/homebrew-core is not on the master branch`? Yep, can do.